dashboard/src/components/LoadingWrapper/LoadingPlaceholder/LoadingPlaceholder.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+import "./LoadingPlaceholder.css";
+
+// Renders a placeholder loader style
+// Based on https://kyusuf.com/post/fake-it-til-you-make-it-css/
+const LoadingPlaceholder: React.SFC = props => {
```

Comment:
You are missing tests, even if it is a simple snapshot one.

Reply:
I saw that the LoadingSpinner didn't have tests, but I can add them for this.

Reply:
Not a lot to test but I guess it's something nice to have